db/seeds.d/75_job_templates.rb

```diff
@@ -1,3 +1,5 @@
+organizations = Organization.unscoped.all
+locations = Location.unscoped.all
```

Comment:
Would it make sense to allow passing in organisation and location IDs and/or even allow this assignment to be optional?